.travis.yml

```diff
@@ -78,16 +80,26 @@ install:
         && git -C occa reset --hard $OCCA_HEAD;
     fi
   - make -C occa -j2
-  - cd $CURR_DIR
-  - export OCCA_DIR=$HOME/install/occa MFEM_DIR=$HOME/install/mfem PETSC_INSTALL=$HOME/install/petsc-3.10.0
+  - export OCCA_DIR=$HOME/install/occa
+  - export PETSC_INSTALL=$HOME/install/petsc-3.10.0
   - test -s "$PETSC_INSTALL/lib/pkgconfig/PETSc.pc"
         || ( curl -O http://ftp.mcs.anl.gov/pub/petsc/release-snapshots/petsc-lite-3.10.0.tar.gz
         && tar xf petsc-lite-3.10.0.tar.gz
+        && rm xf petsc-lite-3.10.0.tar.gz
```

Comment:
`xf` is a mandatory argument for `tar` ("extract" from the "file"), but not relevant to `rm`. PETSc shouldn't be built in the install directory (because we use `make install`) so it doesn't matter whether you delete the tarball.